file-share-service-desktop-client/Modules/Admin/JobViewModels/AppendAclJobViewModel.cs

```diff
@@ -1,24 +1,131 @@
 ﻿using System.Collections.Generic;
 using System.Threading.Tasks;
 using UKHO.FileShareService.DesktopClient.Core.Jobs;
+using UKHO.FileShareAdminClient;
+using UKHO.FileShareAdminClient.Models;
+using System;
+using System.Linq;
+using System.Net;
+using Newtonsoft.Json;
+using Microsoft.Extensions.Logging;
+using Prism.Commands;
+using System.Threading;
+using UKHO.FileShareService.DesktopClient.Core.Models;
 
 namespace UKHO.FileShareService.DesktopClient.Modules.Admin.JobViewModels
 {
     public class AppendAclJobViewModel : BaseBatchJobViewModel
     {
         private readonly AppendAclJob job;
+        private readonly Func<IFileShareApiAdminClient> fileShareClientFactory;
+        private string executionResult = string.Empty;
+        string responseMessage = string.Empty;
+        private bool isCommitting;
+        private bool isExecutingComplete;
+        private readonly ILogger<AppendAclJobViewModel> logger;       
 
-        public AppendAclJobViewModel(AppendAclJob job):base (job)
+        public AppendAclJobViewModel(Func<IFileShareApiAdminClient> fileShareClientFactory, AppendAclJob job, ILogger<AppendAclJobViewModel> logger) : base(job,logger)
```

Comment:
Please position AppendAclJob first in the list of arguments for consistency with other JobViewModels

Reply:
Resolved